src/components/DeckMechanisms/index.js

```diff
@@ -20,6 +20,17 @@ export const FRIENDLY_CHANCES = {
   W16: DAWNSPARKS_HITS * DAWNSPARKS_STAYS,
 }
 
+const FROZEN_ENEMIES_AFTER = {
```

Comment:
This constant name puzzles me.

Reply:
As in "Frozen enemies left after this card's ability has been resolved, in regular RNG mode"

Reply:
That’s a good comment, I suggest we add it. :)

Reply:
Where should comments like that go? I'm not used to adding comments to my js code